src/domain/pricing/PricingPage.tsx

```diff
@@ -205,36 +206,36 @@ const PricingPage: React.SFC<PricingPageProps> = ({
 
   return (
     <div className={styles.pricingPage}>
-      <Text className={styles.fullWidth} as="h2" weight="normalWeight" size="xxl">
-        {t('pricing.heading')}
-      </Text>
-      <Card className={styles.fullWidth}>
-        <CardHeader title={t('pricing.berths.title')} />
-        <CardBody>
-          <Section>{t('pricing.berths.description')}</Section>
-          <Table columns={harborCols} data={berthsData} theme="basic" />
-        </CardBody>
-      </Card>
-      <Card className={styles.fullWidth}>
-        <CardHeader title={t('pricing.winterStorage.title')} />
-        <CardBody>
-          <Section>{t('pricing.winterStorage.description')}</Section>
-          <Table columns={winterStorageCols} data={winterStorageData} theme="basic" />
-        </CardBody>
-      </Card>
-      <Card>
-        <CardHeader title={t('pricing.harborServices.title')} />
-        <CardBody>
-          <Section>{t('pricing.harborServices.description')}</Section>
-          <Table columns={harborServicesCols} data={harborServicesData} theme="basic" />
-        </CardBody>
-      </Card>
-      <Card>
-        <CardHeader title={t('pricing.additionalServices.title')} />
-        <CardBody>
-          <Table columns={additionalServicesCols} data={additionalServicesData} theme="basic" />
-        </CardBody>
-      </Card>
+      <PageTitle title={t('pricing.title')} />
+      <div className={styles.grid}>
+        <Card className={styles.fullWidth}>
```

Comment:
Do we need this outer wrapper `div`? 
Can't we do 
```TSX
   <Card className={classNames(styles.fullWidth, styles.grid)}>
```

Reply:
That would make every card into a grid of its own, the goal here is to have a grid with the cards in it

Reply:
Of course, my bad!